.drone.env

```diff
@@ -1,3 +1,3 @@
 # The version of OCIS to use in pipelines that test against OCIS
-OCIS_COMMITID=ae74e77c4656b9b5fa6af3386a15d10bad49e759
+OCIS_COMMITID=5403f5287551bdf8b089bba3195f1df8e4d6c793
```

Comment:
This commit-id is of the master branch, `ae74e77c4656b9b5fa6af3386a15d10bad49e759` is the latest commit id of `stable-5.0`. Commit id is up-to-date.

Reply:
My bad. I should close this PR then